site/en/guides/pyspinel/install-pyspinel.md

```diff
@@ -0,0 +1,86 @@
+# Install Pyspinel
+
+Your installation options will vary depending on how you use Pyspinel. To review
+packet sniffing options and configurations, refer to the following topics:
+
+*   [Packet Sniffing with Pyspinel](sniffer.md)
+*   [Packet Sniffing using Extcap](sniffer-extcap.md)
```

Comment:
Instead of linking them off to another page entirely before they even do the installation, could we explain the difference between the two so they can decide which they want here, and just link each to the Install sections below?